content/source/layouts/_cloud_content.erb

```diff
@@ -435,6 +435,11 @@
           </li>
           <li>
             <a href="/docs/cloud/api/workspaces.html">Workspaces</a>
+            <ul class="nav">
+              <li>
+                <a href="/docs/cloud/api/workspace-resources.html">Workspace Resources</a>
+              </li>
+            </ul>
           </li>
           <li>
             <a href="/docs/cloud/api/workspace-variables.html">Workspace Variables</a>
```

Comment:
Depending on how conversation goes am happy to move this so it's subheaded under `Workspaces` as well for consistency sake.  Wanted to leave the diversion for now for differences, but also upcoming POCs conversation to be had.

Reply:
Just updating here so anyone coming thru is on the same page also — after speaking with @laurapacilio gonna hold off on reorganizing atm due to current other documentation work.  That said, for the purposes of _Workspace Resources_ it will also not be nested since it sounds like it’ll be easier to start the nesting after said work is completed, so will also persist _Variable Sets_ and _Workspace Specific Variables_ as-is also.  :thumbsup: